### PR TITLE
get started for macvlan

### PR DESCRIPTION
**Pull request by weizhoublue**
---
name: Pull request
about: Tell us about your contribution
labels: ["pr/release/none-required"]
---

---
## Thanks for contributing !

Before submitting a pull request, make sure you read about our Contribution notice here: <https://spidernet-io.github.io/spiderpool/contributing/development/pullrequest/#need-review>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**make sure your commit is signed off**

### CODE REVIEW

File: docs/usage/get-started-macvlan-zh_CN.md [hunk 1]
```diff
@@ -0,0 +1,308 @@
+# 搭建 spiderpool
+
+spiderpool 可用于 underlay 网络场景下提供固定 IP 的解决方案，本文将
+以 [Multus](https://github.com/k8snetworkplumbingwg/multus-cni)、[Macvlan]((https://github.com/containernetworking/plugins/tree/main/plugins/main/macvlan))、
+[Veth](https://github.com/spidernet-io/plugins)、Spiderpool 来来搭建一套完整的 underlay 网络解决方案，该方案能够满足以下各种功能需求：
```
Comment (windsonsea) [inline docs/usage/get-started-macvlan-zh_CN.md:5]:
```suggestion
Spiderpool 可用作 underlay 网络场景下提供固定 IP 的一种解决方案，本文将以
[Multus](https://github.com/k8snetworkplumbingwg/multus-cni)、
[Macvlan]((https://github.com/containernetworking/plugins/tree/main/plugins/main/macvlan))、
[Veth](https://github.com/spidernet-io/plugins) 为例，使用 Spiderpool 搭建一套完整的 underlay 网络解决方案，该方案能够满足以下各种功能需求：
```

File: docs/usage/get-started-macvlan-zh_CN.md [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+
+* 通过简易运维，应用可分配到固定的 underlay IP 地址
+
+* POD 具备多张 underlay 网卡，通达多个 underlay 子网
```
Comment (windsonsea) [inline docs/usage/get-started-macvlan-zh_CN.md:9]:
```suggestion
* Pod 具备多张 underlay 网卡，通达多个 underlay 子网
```
看社区讨论 POD 这种写法是错的，可以用 Pod 或 pod

File: docs/usage/get-started-macvlan-zh_CN.md [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+
+* POD 能够通过 POD IP、 clusterIP、nodePort 等方式通信
```
Comment (windsonsea) [inline docs/usage/get-started-macvlan-zh_CN.md:11]:
```suggestion
* Pod 能够通过 podIP、 clusterIP、nodePort 等方式通信
```
可以全文查找替换一下

File: docs/usage/get-started-macvlan-zh_CN.md [hunk 4]
```diff
@@ -0,0 +1,308 @@
...
+   
+    确认节点上存在 multus 的配置文件 ` ls /etc/cni/net.d/00-multus.conf`
+
+3. 为 macvlan 创建 multus 的 NetworkAttachmentDefinition 配置
+
+需要确认如下配置：
+
+* 确认 macvlan 所需的宿主机父接口，本例子以宿主机 eth0 网卡为例，从该网卡创建 macvlan 子接口给 POD 使用
+
+* 为使用 veth 插件来实现 clusterIP 通信，需确认集群 service 的 clusterIP CIDR， . 例如可基于命令 `kubectl -n kube-system get configmap kubeadm-config -oyaml | grep service-cluster-ip-range` 查询
```
Comment (windsonsea) [inline docs/usage/get-started-macvlan-zh_CN.md:74]:
```suggestion
* 为使用 veth 插件来实现 clusterIP 通信，需确认集群 service 的 clusterIP CIDR，例如可基于命令 `kubectl -n kube-system get configmap kubeadm-config -oyaml | grep service-cluster-ip-range` 查询
```

**Review by windsonsea — COMMENTED** [review]
Some consistency issues

**Comment by ty-dc** [comment]
已在其他 PR 合入，我将关闭该 PR

**Pull request closed without merging**